pentasharp/Models/Entities/TaxiReservations.cs

```diff
@@ -9,7 +10,11 @@ public class TaxiReservations
         public int ReservationId { get; set; }
 
         [Required]
-        public int TaxiId { get; set; }  // Foreign key to Taxi
+        public int? TaxiId { get; set; } = null;
```

Comment:
as per my pervious recommendations on different PRs, remove data annotations, add XML summaries reformat class ctr+a, ctrl+k+f

Reply:
@oltahasimja why is this comment resolved if the proposed fix is not done?